com.unity.netcode.gameobjects/Runtime/NetworkVariable/NetworkVariableBase.cs

```diff
@@ -66,6 +67,7 @@ public void Initialize(NetworkBehaviour networkBehaviour)
         /// Gets or sets the name of the network variable's instance
         /// (MemberInfo) where it was declared.
         /// </summary>
+        [Preserve]
```

Comment:
I second Ian, we should add a changelog entry for this fix but apart from that, it looks good to me!